src/tests/Microsoft.PowerFx.Core.Tests/ExpressionTestCases/Time.txt

```diff
@@ -67,6 +67,12 @@ false
 >> Time(1/0, 2, 3) < Time(8,1,30)
 #Error
 
+>> Date( 2019, 5, 16 ) > Time( 11, 23, 45 )
```

Comment:
Can we add a comment about what does this mean?

Reply:
Unfortunately, we already allowed this in Canvas, and there's not a great way to remove that support.